index.html

```diff
@@ -63,6 +63,224 @@ <h1>I’m Alexandra. Glad to see you!</h1>
         </section>
 
     </div>
+    <!-- MY Works section -->
+    <section class="works-container">
+        <!-- Works section items -->
+
+        <div class="works-grid-item">
+            <div class="template-img flex-item-width">
+                <img src="img/Portfolio.png" alt="">
+            </div>
+            <div class="Tonic flex-item-width">
+                <h2>Tonic</h2>
+            </div>
+            <div class="ol-list flex-item-width">
+                <ol>
+                    <li id="canopy">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly use the `ul` tag instead of the `ol` create listed elements.

## Requireements:
![image](https://user-images.githubusercontent.com/50990925/235325928-ac5767e4-f396-425f-a2c9-278729659cf6.png)